docs/writing-content.md

```diff
@@ -0,0 +1,52 @@
+# Writing content in Nodejs.dev
+
+We use Markdown to write content for nodejs.dev. We follow the [CommonMark](https://commonmark.org/) specification. And you can use MDX to write React components in Markdown.
+
+## Special syntax
+
+### Code blocks
+
+> **Note:** we use single cote in examples to avoid conflicts with Markdown syntax.
+
+To write simple code use this syntax:
+
+```md
+    '''js
```

Comment:
You should do:

`````md
````md
  ```js
    const foo = 'bar';
  ```
````
`````

Reply:
I didn't know this way of writing at all thanks.